internal/buildscripts/packaging/fpm/Dockerfile

```diff
@@ -12,5 +12,6 @@ ENV PACKAGE="deb"
 ENV VERSION=""
 ENV ARCH="amd64"
 ENV OUTPUT_DIR="/repo/dist/"
+ENV SMART_AGENT_RELEASE="latest"
```

Comment:
If we wanted to keep in step with our agent dep can we use the output of `go list -m github.com/signalfx/signalfx-agent` instead of pulling from latest?

Reply:
per offline convo: pinning to v5.9.0 for now and using `20210308172704-d05049330d0d` in go.mod is good workaround (though looks like there's a 5.9.1 release on the way)